src/pages/journal/components/vehicle-details/__tests__/vehicle-details.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { TestBed, async, ComponentFixture } from '@angular/core/testing';
+import { IonicModule } from 'ionic-angular';
+import { VehicleDetailsComponent } from '../vehicle-details';
+import { By } from '@angular/platform-browser';
+
+
+describe('VehicleDetailsComponent', () => {
+  let fixture: ComponentFixture<VehicleDetailsComponent>;
+  let component: VehicleDetailsComponent;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        VehicleDetailsComponent
+      ],
+      imports: [IonicModule],
+
+    }).compileComponents().then(() => {
+      fixture = TestBed.createComponent(VehicleDetailsComponent);
+      component = fixture.componentInstance;
+    })
+  }));
+
+  describe('Class', () => {
+    it('should create', () => {
+      expect(component).toBeDefined();
+    });
+  });
+
+  describe('DOM', () => {
+    it('should show vehicle details when showVehicleDetails is true', () => {
```

Comment:
This test name doesn't look quite right. You're setting `showDimensions`.

Reply:
fixed :)